src/main/scala/io/opentargets/etl/backend/Configuration.scala

```diff
@@ -42,7 +42,7 @@ object Configuration extends LazyLogging {
 
   case class InputInfo(format: String, path: String)
   case class InputExtension(extensionType: String, path: String) {
-    require(path.endsWith("json"))
+    require(path.endsWith("json") || path.endsWith("jsonl"))
```

Comment:
@JarrodBaker you should not need this. either algebraic type of build extractors or just deal with it later on as you are doing but it looks like double effort